python_modules/dagster/dagster/_core/definitions/freshness_checks/non_partitioned.py

```diff
@@ -59,6 +60,8 @@ def build_freshness_checks_for_non_partitioned_assets(
             (cron provided).
         freshness_cron (Optional[str]): The check will pass if the asset was updated within
             maximum_lag_minutes of the most recent tick of this cron.
+        timezone (Optional[str]): The timezone to use for the cron schedule. If a freshness_cron is
+            provided, a timezone must also be provided.
```

Comment:
I think we can default to UTC like we do in the other places where crons are provided, no?

Reply:
I thought we discussed this and decided not to provide a default. Open to changing though

Reply:
My interpretation was that we decided not to default it to the same timezone as the TimeWindowPartitionsDefinition.

Reply:
you're right. I think I misremembered this as not having any default.